common/azure/src/main/kotlin/com/cosmotech/api/azure/adx/AzureDataExplorerClient.kt

```diff
@@ -24,6 +36,7 @@ import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.stereotype.Service
 
 private const val REQUEST_TIMEOUT_SECONDS = 30L
+private const val SLEEPING_TIME: Long = 5000
```

Comment:
This parameter should be configurable. Could you please add a property (in csmPlatformProperties) in order to change it without deployment?